frontend/src/validation/validators.ts

```diff
@@ -0,0 +1,73 @@
+import { ethers } from 'ethers';
+
+import type { Chain } from '@/types/data';
+import type { TokenAmount } from '@/types/token-amount';
+
+/**
+ * Checks if the provided `value` is a valid ethereum address
+ *
```

Comment:
True, got carried away at the first implementation try since the code was not really speaking for itself. 
Now that i refactored them to a state where its obvious whats happening i will remove the comments and type the return values.
Thanks for the tip!